packages/userstorage/src/browser/user-storage-service.ts

```diff
@@ -17,5 +18,5 @@ export interface UserStorageService extends Disposable {
 }
 
 export interface UserStorageChangeEvent {
-    uris: string[];
+    uris: URI[];
```

Comment:
Oh my bad, this should be changed also (you mean the uri arguments as URI instead of strings right?). But don't you think this also makes sense for the event?

Reply:
for events ok also, just don't forget to change comparison in the resource